frontend/integration-tests/tests/olm/crd.scenario.ts

```diff
@@ -0,0 +1,45 @@
+import { browser, $, $$, by, ExpectedConditions as until, element } from 'protractor';
+import { appHost, checkLogs, checkErrors } from '../../protractor.conf';
+
+describe('Test for CRD related pages', () => {
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  it('Check CatalogSource instances', async() => {
```

Comment:
for the description in `describe` and `it`, it would be better if we can make it clear

Reply:
Also, I don't see a reason why we add `crd.scenario.ts` into `olm directory`

Reply:
This is related to olm for CRD instances check.

Reply:
> for the description in `describe` and `it`, it would be better if we can make it clear

Updated.

Reply:
> Also, I don't see a reason why we add `crd.scenario.ts` into `olm`

This is related to olm for CRD instances check.

Reply:
ok, looks good to me